app/views/devise/shared/_social_login.html.haml

```diff
@@ -1,15 +1,16 @@
 #social-login-section
-  - if (Rails.application.config.omniauth_google_key.present? && Rails.application.config.omniauth_google_secret.present?) || (Rails.application.config.omniauth_facebook_key.present? && Rails.application.config.omniauth_facebook_secret.present?)
+  - if Site.current.social_login_enabled?
```

Comment:
We have a [`current_site`](https://github.com/mconf/mconf-web/blob/b7095825ee2fbccffae0d38b58b2e84e7196f1da/app/controllers/application_controller.rb#L58) helper that caches the site, it's usually better to use it in views.